src/js/lib/logging/test/test.js

```diff
@@ -0,0 +1,79 @@
+const assert = require('assert');
+const axios = require('axios').default
+const fs = require('fs')
+const mockFs = require('mock-fs');
+const path = require('path');
+
+const { startServer } = require('../server')
+
+
+describe('Logging Server', function () {
+
+    let server
+    let serverUrl
+
+    const dummyMessage = '<Dummy message>'
+    const goodConfig = {
+        headers: {
+            'Content-Type': 'text/plain',
+            'Origin': 'http://localhost/'
+        }
+    }
+    const wrongOriginConfig = {
+        headers: {
+            'Content-Type': 'text/plain',
+            'Origin': 'http://attacker/'
+        }
+    }
+    const wrongContentTypeConfig = {
+        headers: {
+            'Content-Type': 'image/jpeg',
+            'Origin': 'http://localhost/'
+        }
+    }
+
+    beforeEach(function (done) {
```

Comment:
The name of this function does not really tell what does it do. The names should be more semantic.

Reply:
Ah, this is inside of tests, ignore me here, sorry! It's ok!